packages/dnb-eufemia/src/components/accordion/AccordionDocs.ts

```diff
@@ -0,0 +1,155 @@
+import { PropertiesTableProps } from '../../shared/types'
+
+export const accordionProperties: PropertiesTableProps = {
+  id: {
+    doc: 'a unique `id` that will be used on the button element. If you use `remember_state`, an id is required.',
```

Comment:
```suggestion
    doc: 'A unique `id` that will be used on the button element. If you use `remember_state`, an id is required.',
```

Good if we rule the docs the same and start capitalized on every single doc property.

Reply:
Will fix the capitalization!